extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSTable.scala

```diff
@@ -45,6 +45,10 @@ class TPCDSTable(tbl: String, scale: Int, options: CaseInsensitiveStringMap)
   // https://www.tpc.org/tpc_documents_current_versions/pdf/tpc-ds_v3.2.0.pdf
   val useTableSchema_2_6: Boolean = options.getBoolean("useTableSchema_2_6", true)
 
+  // #2543, one split represents the number of rows generated per second
+  // in TPCDSTableGenerateBenchmark
+  val splitPerTask: Int = options.getInt("splitPerTask", 1)
```

Comment:
The default value 1 is too small, maybe 30 is more reasonable, or judge the default value by scale